lib/osf-components/addon/components/ancestry-display/template.hbs

```diff
@@ -3,12 +3,21 @@
         <span local-class='AncestorTitle'>
             {{#if (and this.useLinks ancestor.id)}}
                 <OsfLink
-                    data-test-ancestor-title={{index}}
+                    data-test-ancestor-link={{index}}
```

Comment:
It's worth checking with QA to make sure they're not using this before it's changed.

Reply:
Checked with QA and these selectors are not currently being used